transformer_engine/pytorch/tensor/mxfp8_tensor.py

```diff
@@ -262,6 +262,39 @@ def clone(self) -> MXFP8Tensor:
             },
         )
 
+    def empty_like(self, *args, **kwargs):
+        """Create a new empty tensor with the same shape and type as this tensor"""
+        new_rowwise_data = (
+            torch.empty_like(self._rowwise_data, *args, **kwargs)
+            if self._rowwise_data is not None
+            else None
+        )
+        new_columnwise_data = (
+            torch.empty_like(self._columnwise_data, *args, **kwargs)
+            if self._columnwise_data is not None
+            else None
+        )
+        new_rowwise_scale_inv = (
+            torch.empty_like(self._rowwise_scale_inv, *args, **kwargs)
```

Comment:
Same as in the blockwise, but here especially so because in MXFP8 case the scale inverses have to be zero-initialized.